naming/dns_resolver_18_test.go

```diff
@@ -0,0 +1,33 @@
+// +build go1.8
+
+package naming
+
+import (
+	"context"
+	"fmt"
+	"net"
+)
+
+func replaceNetFunc() func() {
+	oldLookupHost := lookupHost
+	oldLookupSRV := lookupSRV
+	targetTc = fakeTargetTc
+	lookupHost = func(ctx context.Context, host string) ([]string, error) {
```

Comment:
please define this func next to hostLookupTbl and share with the other version-specific file.